src/loaders/styleguide-loader.ts

```diff
@@ -45,6 +46,9 @@ export function pitch(this: Rsg.StyleguidistLoaderContext) {
 
 	const config = this._styleguidist;
 
+	// get highlight code languages. Array of languages that allow you to decide which syntax highlighter to add
+	getHighlightCodeLanguages(config.highlightCodeLanguages);
```

Comment:
The name of the function suggests that it returns a list of languages which isn't used.

Maybe:

```suggestion
	loadCodeHighlightLanguages(config.highlightCodeLanguages);
```